index.js

```diff
@@ -191,6 +192,42 @@ demo`;
   info: () => {
     info();
   },
+  config: () => {
+    const args = arg({});
+
+    const action = args._[1];
+    const key = args._[2];
+    const value = args._[3];
+
+    if (!action.length) {
+      return invalid("please provide the action to perform on the config");
+    }
+
+    switch (action) {
+      case "set":
+        if (!key) {
+          return invalid("Please specific the config key to set.");
+        }
+        if (!value) {
+          return invalid("Please specific the config value to set.");
```

Comment:
should it say `specify`?
```suggestion
          return invalid("Please specify the config key to set.");
        }
        if (!value) {
          return invalid("Please specify the config value to set.");
```